market/connector/docusign/demo/eSignatureDemo/src/net/docusign/auth/UserUriFilter.java

```diff
@@ -73,9 +73,9 @@ private static URI routeToUserUri(URI uri, JsonNode accounts)
     
     JsonNode first = accounts.get(0);
     URI baseUri = URI.create(first.get("base_uri").asText());
-    URI userUri = UriBuilder.fromUri(baseUri).path("/restapi/v2.1/accounts/{myId}/{resource}")
+    URI userUri = UriBuilder.fromUri(baseUri).path("/restapi/v2.1/accounts/{myId}")
       .resolveTemplate("myId", first.get("account_id").asText())
-      .resolveTemplate("resource", resource)
+      .path(resource)
```

Comment:
why you changed this? or other question why you don't changed both (myId too) 😄

Reply:
my-id is a single part within an URI /bla/{myId}/blu and can therefore be correctly replaced with the templating mechanism. It's build just for that.
With the resource it's different ... this is the first time I had to access resources which do not only refernce a single resource /envelopes ... but also sub-resource /envelopes/234243/documents
So the URI builder created an invalid URI for these. Which is now no longer an issue.